server/src/test/java/com/broadcom/lsp/cobol/usecases/TestExtraneousInputEOFExpecting.java

```diff
@@ -35,19 +35,21 @@
           + "           if (1 > 0) NEXT SENTENCE{|1}"; // No dot at the end of file
 
   private static final String MESSAGE =
-      "Extraneous input '<EOF>' expected {ACCEPT, ADD, ALTER, CALL, CANCEL, CLOSE, "
```

Comment:
same message but the order of tokens is changed. Earlier implementation, the order of token was sorted. Need to check this. Same for rest of the PR.